Plugin: Hypixel数据查询

**插件名称：**

Hypixel数据查询

**插件功能：**

查询Hypixel的数据

**PyPI 项目名：**

nonebot-plugin-hypixel

**插件 import 包名：**

nonebot_plugin_hypixel

**插件项目仓库/主页链接：**

https://github.com/SkyDynamic/nonebot_plugin_hypixel

**标签：**

[{"label":"MC","color":"#6fea52"},{"label":"Hypixel","color":"#d5ea52"},{"label":"Hyp","color":"#d5ea52"}]

# 📃 商店发布检查结果

> Plugin: Hypixel数据查询

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot_plugin_hypixel 的信息如下：
    name         : nonebot-plugin-hypixel                                  
     version      : 1.0.0                                                   
     description  : 基于OneBot适配器的NoneBot2获取Mineceaft最大的小游戏服务器Hypixel的玩家数据的插件 
    
    dependencies
     - nonebot-adapter-onebot &gt;=2.1.5
     - nonebot2 &gt;=2.0.0rc1
插件 nonebot_plugin_hypixel 加载出错：
01-06 06:54:59 [SUCCESS] nonebot | NoneBot is initializing...
01-06 06:54:59 [INFO] nonebot | Current Env: prod
01-06 06:55:00 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_hypixel&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_hypixel&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_hypixel/__init__.py&quot;, line 8, in &lt;module&gt;
    from .request import PlayerNameNotFound, HypixelAPICallError, player_data
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_hypixel/request.py&quot;, line 2, in &lt;module&gt;
    from httpx import AsyncClient
ModuleNotFoundError: No module named &#x27;httpx&#x27;</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 标签: MC-#6fea52, Hypixel-#d5ea52, Hyp-#d5ea52。</li><li>✅ 项目 <a href="https://github.com/SkyDynamic/nonebot_plugin_hypixel">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot_plugin_hypixel/">nonebot_plugin_hypixel</a> 已发布至 PyPI。</li></code></pre></details>

---

💪 Powered by NoneBot2 Publish Bot
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -3959,5 +3959,28 @@
       }
     ],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_hypixel",
+    "project_link": "nonebot-plugin-hypixel",
+    "name": "Hypixel数据查询",
+    "desc": "查询Hypixel的数据",
+    "author": "SkyDynamic",
+    "homepage": "https://github.com/SkyDynamic/nonebot_plugin_hypixel",
+    "tags": [
+      {
+        "label": "MC",
+        "color": "#6fea52"
+      },
+      {
+        "label": "Hypixel",
+        "color": "#d5ea52"
+      },
+      {
+        "label": "Hyp",
+        "color": "#d5ea52"
+      }
+    ],
+    "is_official": false
   }
 ]

